### ISSUE TITLE
Fix 'test_scan_results/test_redhat_duplicate_vulns' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1569]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team: "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

## Case:

|  Type | Description  |
|--|--|
|Test Path | test_scan_results/test_redhat_duplicate_vuln
| Consistent | yes
| Test Execution | 3/3 executions failed
| Cases | RedHat_6_feed      
|| RedHat_7_feed         
|| RedHat_8_feed
| Cases Fails | 3
| Summary | On `test_redhat_duplicate_vulns` when `check_log_event` is called. it requires research to know if there is a problem with parser error or if we don't detect log.
| Error message | "Could not find the log event: Starting preparse step of feed 'RHEL8'"

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix 
- [ ] Full Green/ Full Yellow  in `test_scan_results/test_redhat_duplicate_vuln` for 3 times. 
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by mdengra** [comment]
### 2021-07-16

Test results with the modules `sca`, `syscollector` and `rootcheck` modules disabled in the `ossec.conf`: 

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_redhat_duplicate_vulns_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6828786/test_redhat_duplicate_vulns_local_dm_r1.log)| 2021-07-16 | Miguel | :green_circle: 
|[test_redhat_duplicate_vulns_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6828789/test_redhat_duplicate_vulns_local_dm_r2.log)| 2021-07-16 | Miguel | :yellow_circle: 
|[test_redhat_duplicate_vulns_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6828791/test_redhat_duplicate_vulns_local_dm_r3.log)| 2021-07-16 | Miguel | :yellow_circle:

**Comment by damarisg** [comment]
This issue is closed by #1605

**Comment by damarisg** [comment]
This issue is closed by #1605 

## Test Results from `1531-full-yellow-vuln-det`

| Test Executions | Tests | Date  | By  | Status | 
|--|--|--|--|--|
|[YellowResultsScanRedhatDuplicateVuln.log](https://github.com/wazuh/wazuh-qa/files/6858256/YellowResultsScanRedhatDuplicateVuln.log)| test_scan_results/test_redhat_duplicate_vuln |2021-07-21| Seyla| 🟢 
|[YellowResultsScanRedhatDuplicateVuln2.log](https://github.com/wazuh/wazuh-qa/files/6858254/YellowResultsScanRedhatDuplicateVuln2.log) | test_scan_results/test_redhat_duplicate_vuln|2021-07-21| Seyla| 🟢
| [YellowResultsScanRedhatDuplicateVuln3.log](https://github.com/wazuh/wazuh-qa/files/6858255/YellowResultsScanRedhatDuplicateVuln3.log)|test_scan_results/test_redhat_duplicate_vuln| 2021-07-21| Seyla|  🟢

| Reference | Status | 
|--|--|
| 🟢 | Pass without Warnings |

**Comment by damarisg** [comment]
Open because it is affect by #1602

**Comment by damarisg** [comment]
This issue is closed because the test was disabled by #1602.
